packages/core/src/browser/cookie.ts

```diff
@@ -1,29 +1,25 @@
 import { display } from '../tools/display'
 import { ONE_MINUTE, ONE_SECOND } from '../tools/utils/timeUtils'
-import { findCommaSeparatedValue, generateUUID } from '../tools/utils/stringUtils'
+import { generateUUID } from '../tools/utils/stringUtils'
 
 export interface CookieOptions {
   secure?: boolean
   crossSite?: boolean
   domain?: string
 }
 
-export function setCookie(name: string, value: string, expireDelay: number, options?: CookieOptions) {
-  const date = new Date()
-  date.setTime(date.getTime() + expireDelay)
-  const expires = `expires=${date.toUTCString()}`
-  const sameSite = options && options.crossSite ? 'none' : 'strict'
-  const domain = options && options.domain ? `;domain=${options.domain}` : ''
-  const secure = options && options.secure ? ';secure' : ''
-  document.cookie = `${name}=${value};${expires};path=/;samesite=${sameSite}${domain}${secure}`
+const cookieMap = new Map<string, string>()
+
+export function setCookie(name: string, value: string, _expireDelay: number, _options?: CookieOptions) {
+  cookieMap.set(name, value)
```

Comment:
This is the actual change of the library.
We are replacing cookies with a local map, so that we are GDPR conform.